kevm-pyk/src/kevm_pyk/__main__.py

```diff
@@ -216,6 +223,26 @@ def exec_foundry_kompile(
             md_selector=md_selector,
             profile=profile,
         )
+    kevm = KEVM(foundry_definition_dir, main_file=foundry_main_file, profile=profile)
+    if regen or rekompile or reparse or not parsed_spec.exists():
+        prove_args = add_include_arg(includes)
+        kevm.prove(
+            foundry_main_file,
+            spec_module_name=spec_module,
+            dry_run=True,
+            args=(['--emit-json-spec', str(parsed_spec)] + prove_args),
+        )
+    if regen or rekompile or reparse or reinit or not kcfgs_file.exists():
+        cfgs: Dict[str, Dict] = {}
+        for module in read_kast_flatmodulelist(parsed_spec).modules:
+            for claim in module.claims:
+                cfg_label = claim.att["label"]
+                _LOGGER.info(f'Producting KCFG: {cfg_label}')
+                cfgs[cfg_label] = KCFG_from_claim(kevm.definition, claim).to_dict()
+        with open(kcfgs_file, 'w') as kf:
+            kf.write(json.dumps(cfgs))
```

Comment:
What is the advantage of having a single file for all the specs? (I guess we have a single main file because kompiling all modules at once is more efficient than kompiling them separately. But for the specs it shouldn't matter, right?)